tests/conftest.py

```diff
@@ -164,7 +170,9 @@ def mock_asset(db):
         users=[User.get_users_by_course_id(course_id=course.id)[0]],
     )
     std_commit(allow_test_environment=True)
-    return asset
+    yield asset
+    db_session.delete(asset)
+    std_commit(allow_test_environment=True)
```

Comment:
We delete the mock asset after the example has run, so that repeatedly calling the fixture doesn't become a magic porridge pot.